src/rsk/rskEngine.js

```diff
@@ -592,6 +582,13 @@ export class RskEngine extends CurrencyEngine {
               this.log(msg)
               throw new Error(msg)
             }
+            // Convert hex
+            result.result = result.result.replace('0x', '')
```

Comment:
Remove this line. Just use isHex as that should take care of the check

Reply:
It fails without that. Maybe I can remove the hex validation directly...